autonomy/cli/scaffold_fsm_templates/components.py

```diff
@@ -70,10 +70,9 @@ class {RoundCls}({ABCRoundCls}):
         \"\"\"{RoundCls}\"\"\"
 
         {todo_abstract_round_cls}
-        # TODO: set the following class attributes
         round_id: str = "{round_id}"
-        allowed_tx_type: Optional[TransactionType]
-        payload_attribute: str = {PayloadCls}.transaction_type
+        allowed_tx_type: Optional[TransactionType] = {PayloadCls}.transaction_type
+        payload_attribute: str = "{round_id}"
```

Comment:
this payload attribute is weird and will be remove entirely during refactoring before next major release